### ISSUE TITLE
[Help Wanted]: Major packages in the x11-repo need updates

### ISSUE THREAD

**Issue by thunder-coding** [termux/x11-packages#613]
Outdated package include whole of "mate family" of packages, mesa, "qt family". Some of the packages are outdated since years and even dail to build with latest Android NDK. Updating them will certainly improve the situation.

**Comment by Yisus7u7** [comment]
> Outdated package include whole of "mate family" of packages, mesa, "qt family". Some of the packages are outdated since years and even dail to build with latest Android NDK. Updating them will certainly improve the situation.

Yes, I have realized that, now that I have time I will see that I will be updating

**Comment by Yisus7u7** [comment]
#615

**Comment by ar664** [comment]
#632
